server/code/setup/cpdb

```diff
@@ -17,7 +17,7 @@
 """ A CLI utility for managing the Candlepin database. """
 
 
-import commands
+import subprocess
```

Comment:
`subprocess` doesn't have `getstatusoutput` on Python2, so you'd need something like:

```python
try:
    from commands import getstatusoutput
except ImportError:
    from subprocess import getstatusoutput
```